deploy/clowdapp.yaml

```diff
@@ -83,13 +83,13 @@ objects:
                   optional: true
             - name: INSIGHTS_CONTENT_SERVICE__LOGGING__LOGGING_TO_SENTRY_ENABLED
               value: ${SENTRY_ENABLED}
-            - name: INSIGHTS_CONTENT_SERVICE__SENTRY_DSN
+            - name: INSIGHTS_CONTENT_SERVICE__SENTRY__DSN
```

Comment:
please make sure you test the loading of these environment variable locally before adding them to the clowdapp.

Reply:
Tested and also tested in ephemeral